packages/scripts/scripts/check-licenses.js

```diff
@@ -0,0 +1,208 @@
+/**
+ * External dependencies
+ */
+const spawn = require( 'cross-spawn' );
+const { existsSync, readFileSync } = require( 'fs' );
+const chalk = require( 'chalk' );
+
+/**
+ * Internal dependencies
+ */
+const { hasCliArg } = require( '../utils' );
+
+const ERROR = chalk.reset.inverse.bold.red( ' ERROR ' );
+
+const prod = hasCliArg( '--prod' ) || hasCliArg( '--production' );
+const dev = hasCliArg( '--dev' ) || hasCliArg( '--development' );
+const gpl2 = hasCliArg( '--gpl2' );
+
+const gpl2Licenses = [
```

Comment:
Nitpick here, but it'd be nice to call this `gpl2CompatibleLicenses` or something like that.

Reply:
👍🏻